src/vm/fieldmarshaler.h

```diff
@@ -230,7 +259,7 @@ VOID FmtValueTypeUpdateCLR(LPVOID pProtectedManagedData, MethodTable *pMT, BYTE
     }
 
 #define ELEMENT_SIZE_IMPL(NativeSize, AlignmentReq)     \
-    UINT32 NativeSizeImpl() const                       \
+    UINT32 NativeSizeCoreImpl() const                       \
```

Comment:
What does the "Core" in the name stand for? It looks odd to have `NativeSizeCoreImpl` when there is no `NativeSizeImpl` anymore.

Also, some comments still refer to `NativeSizeImpl`.

Reply:
I've refactored the FieldMarshaler "pseudo-vtable" generation to support generating `NativeSize` without having to use `NativeSizeCore/NativeSizeCoreImpl`.